File: BaekJoon/BasicMath2/1929.py
import math


M, N = map(int, input().split())

def isPrime(n):
    if n == 1:
        return False
    for i in range(2, math.floor(math.sqrt(n))+1):
        if n%i == 0:
            return False
    return True

for n in range(M, N+1):
    if isPrime(n) == True:
        print(n)

File: BaekJoonSwift/BaekJoonSwift/Basic Math2/1929.swift
import Foundation

func isPrime(num : Int) -> Bool {
    var result = true
    if num == 1 {
        return false
    }
    
    if num == 2 || num == 3 {
        return true
    }
    
    for i in 2...Int(sqrt(Double(num))) {
        if num % i == 0 {
            result = false
            break
        }
    }
    
    return result
}

let values = readLine()!.split(separator: " ").map { Int($0)! }

for i in values[0]...values[1] {
    if isPrime(num: i){
        print(i)
    }
}
